kyuubi-server/src/main/scala/org/apache/kyuubi/operation/KyuubiOperation.scala

```diff
@@ -156,8 +157,11 @@ abstract class KyuubiOperation(session: Session) extends AbstractOperation(sessi
   override def shouldRunAsync: Boolean = false
 
   override def setState(newState: OperationState): Unit = {
+    MetricsSystem.tracing { ms =>
+      ms.decCount(MetricRegistry.name(OPERATION_STATE, opType, state.toString.toLowerCase))
+      ms.incCount(MetricRegistry.name(OPERATION_STATE, opType, newState.toString.toLowerCase))
+      ms.markMeter(MetricRegistry.name(OPERATION_STATE, newState.toString.toLowerCase))
```

Comment:
what is this for?

Reply:
I think we can remove this one, it seems meaning less and always increasing.

Reply:
what is different from L43

Reply:
the state value is different with that of L43.

Reply:
i see